src/redux/getters/getTripTimes.js

```diff
@@ -1,5 +1,5 @@
 import { getTripEvents } from './getEvents';
-import { convertAllDates } from './convertDate';
+//import { convertAllDates } from './convertDate';
```

Comment:
@gzinck will wait for you to merge latest PR and then resolve merge conflicts before I do anything with this branch

Reply:
Sounds good!